### ISSUE TITLE
Fix flex search

### ISSUE THREAD

**Issue by MattDodsonEnglish** [TrueBlocks/trueblocks-docs#49]
That's an important part of a website.

It happened when dependabot upgraded a package. Getting this traceback:

```shell
render of "page" failed: execute of template failed:
template: _default/single.html:28:7: executing "_default/single.html" at <partial "footer/script-footer.html" .>:
error calling partial:
"/trueblocks-docs/node_modules/@hyas/doks/layouts/partials/footer/script-footer.html:37:23": execute of template failed:
template:
partials/footer/script-footer.html:37:23: executing "partials/footer/script-footer.html" at <append $flexSearch>: error calling append: reflect: call of reflect.Value.Type on zero Value
```

**Comment by MattDodsonEnglish** [comment]
I thought it was a problem of having a version that was too new. 
1f7435485de263771423ab4df9fad77a87175803

But I see that mainline has upgraded, so it must be something else.

I would bet it's about this:
`executing "partials/footer/script-footer.html" at <append $flexSearch>: error calling append: reflect: call of reflect.Value.Type on zero Value`

**Comment by MattDodsonEnglish** [comment]
I think this should be fixed, with this commit 743c3ccbde3490621b38915eea85eb5c654f598e and it's parent.

However, the search is not very good though. It seems to look through only docs, and doesn't return much. I see that the mainline doks site has nice search, but they make a third party call to algolia. Maybe that's not a good solution for our site, and a different search should be implemented.
